contrib/intel_env.sh

```diff
@@ -32,7 +32,8 @@ export BREAKDANCER_PATH=$GENOMICS_DIR/ohsu/dnapipeline/breakdancer-1.4.5-unstabl
 export PINDEL_PATH=$GENOMICS_DIR/ohsu/dnapipeline/pindel/
 export SPARK_HOME=/opt/cloudera/parcels/CDH/lib/spark
 export ADAM_HOME=/opt/adam/adam-distribution-0.16.0
-export PATH=$PINDEL_PATH:$BREAKDANCER_PATH:$TOPHAT_EXE_PATH:$CUFFLINKS_TOOLS_EXE_PATH:$BOWTIE_TOOLS_EXE_PATH:$GENOMICS_DIR:$BWA_DIR:$SAMTOOLS_DIR:$SAMTOOLS_v020_EXE_PATH:$PICARD_PATH:$PYTHON_2_7_PATH/bin:$SPARK_HOME/bin:$ADAM_HOME/bin:$PATH
+export SAMBAMBA_PATH=$GENOMICS_DIR/ohsu/dnapipeline/sambamba-0.5.8/
```

Comment:
Need to add these variables to one more file: ccc_poc_env.sh where GENOMICS_DIR=/opt/sync/Genomics to reflect the PoC setup.